ush/fv3gfs_driver_grid.sh

```diff
@@ -171,26 +171,26 @@ if [ $gtype = uniform ] || [ $gtype = stretch ] || [ $gtype = nest ];  then
     touch $TMPDIR/orog.file1
     tile=1
     while [ $tile -le $ntiles ]; do
-      echo "$script_dir/fv3gfs_make_orog.sh $res $tile $grid_dir $orog_dir $script_dir $topo $TMPDIR " >>$TMPDIR/orog.file1
+      echo "$script_dir/fv3gfs_make_orog.sh $res $tile $grid_dir $orog_dir $topo $add_lake $TMPDIR " >>$TMPDIR/orog.file1
       tile=$(( $tile + 1 ))
     done
     aprun -j 1 -n 4 -N 4 -d 6 -cc depth cfp $TMPDIR/orog.file1
```

Comment:
The tile loop was moved to 'fv3gfs_make_orog.sh'.  So this 'cfp' logic for WCOSS-Cray no longer works.